### DISCUSSION TITLE
Interest in paid hosting

### DISCUSSION THREAD

**Discussion by sigaloid** [TriliumNext/Trilium#2067] [General]
Just curious how many people would be interested in a paid service to sync an existing notebook to (and have web access via a subdomain, like "username.trilium.host" or similar) or start a new notebook. Maybe $1-3 a month depending on how many sign up and on database size. It's a niche application so I'm curious to gauge interest in it before setup. Thanks!

**Comment by davidmbesonen** [comment]
I'm not sure how interested I'd be (I haven't committed myself to using Trilium yet).  

However, to whatever degree I was interested, baseline functionality would need to include server-side encryption.  (Maybe this is already part Trilium's feature set?)

**Reply by sigaloid** [reply]
You can "protect" a subtree thus end to end encryption. https://github.com/zadam/trilium/wiki/Protected-notes

**Reply by rrouviere** [reply]
I would absolutely be interested in server-side encryption.
I think however that it could be implemented using [nested note encryption (using different key)](https://github.com/zadam/trilium/issues/2143#issuecomment-919458868), with a GUI option to locally (ie. outside of the document itself) store the password/key and decrypt it. The fact that attributes aren't protected might be a problem.
It would be more flexible as well, since you could decide for example to store locally the key for the "Root" note in every machine except the server, the key for the note "Root/Finances" only on machine A, the key for "Root/Whatever" only on machine B...
Though of course it would be great to be able to relax protected mode restrictions and features (shadows in the title, timeout, search restriction...) in this use case.

As for paid hosting, and except for multiuser features,I honestly think there's much point, since it's very easy to setup a docker container and that most trilium users are probably capable of doing it.
